torch_glow/src/PyTorchModelLoader.cpp

```diff
@@ -314,6 +314,35 @@ void PyTorchModelLoader::loadConvolution(const torch::jit::Node *ptNode) {
   addGlowNodeValue(outputs[0], output->getResult());
 }
 
+void PyTorchModelLoader::loadLinear(const torch::jit::Node *ptNode) {
+  auto inputs = ptNode->inputs();
+  auto outputs = ptNode->outputs();
+  assert(inputs.size() == 3);
+  assert(outputs.size() == 1);
+
+  // Indexes of aten::linear inputs
+  struct Inputs {
+	enum {
+	  input = 0,
+	  weights = 1,
+	  bias = 2,
+	}
+  }
+
+  glow::NodeValue input = getGlowNodeValue(inputs[Inputs::input]);
+  glow::NodeValue weights = getGlowNodeValue(inputs[Inputs::weights]);
+  glow::NodeValue bias = getGlowNodeValue(inputs[Inputs::bias]);
+
+  glow::NodeValue output;
+
+  // add some shape check here
+  // auto *broadcasted_bias = f_->createBroadcast(bias, {input.dims()});
+  auto *mul = f_->createMatmul("mul", input, weights);
```

Comment:
This is basically an FC right? Can we just load this as an FC? Otherwise what is the point of fusing these together in the JIT IR?